doc/nrf/known_issues.rst

```diff
@@ -927,22 +927,22 @@ tx_buffer_length set incorrectly
 Zephyr
 ******
 
-.. rst-class:: v1-4-99-dev1
+.. rst-class:: v1-5-0
 
 Disabling USB causes crash
   Disabling USB by calling :c:func:`usb_disable` with a subsequent call to :c:func:`usb_dc_detach` tries to free unallocated memory and crashes.
 
```

Comment:
```suggestion
```
This issue only affected v1-4-99-dev1

Reply:
Removed from the known issues, as v1.5.0 replaces v1.4.99.